vllm/model_executor/layers/triton_kernel/prefix_prefill.py

```diff
@@ -5,6 +5,8 @@
 import triton
 import triton.language as tl
 
+TESLA = 'Tesla' in torch.cuda.get_device_name(0)
```

Comment:
would it be possible to check for compute capability instead? also, we should do this inside `context_attention_fwd`, as calling CUDA APIs before we set `CUDA_VISIBLE_DEVICES` will lead to errors.

Reply:
Maybe we can set `--prefix-block-size` as a parameter in `CacheConfig`?

Reply:
this sort of a thing should be ideally derived automatically.

Reply:
@Yard1 @caoshiyi Does the block size affect the memory utilization or prefix speed?

Reply:
@esmeetu The block size is mainly dependent on the shared mem size for different GPU architectures. It will affect the prefix-prefill kernel speed a little bit but has nothing to do with the GPU memory utilization.